gno.land/pkg/gnoweb/components/view_source.go

```diff
@@ -1,16 +1,29 @@
 package components
 
+import (
+	"fmt"
+)
+
 const SourceViewType ViewType = "source-view"
 
+type DisplayMode int
+
+const (
+	ModeCode DisplayMode = iota
+	ModeMarkdown
+)
```

Comment:
No, if you are using this within the template, it makes no sense to use int here. Just change this to string, or find a way to expose this as const within the template. But if it's not something simple, simply use string.

```suggestion
type DisplayMode string

const (
	ModeCode DisplayMode = "code"
	ModeMarkdown = "markdown"
)
```